frontend/src/components/Forms/TeamEventCreditsForm/TeamEventCreditsForm.tsx

```diff
@@ -225,8 +238,21 @@ const TeamEventCreditForm: React.FC = () => {
         </div>
 
         <div style={{ margin: '2rem 0' }}>
-          <label style={{ fontWeight: 'bold' }}>Pending Approval For:</label>
-          <p>{`[${pendingTEC}]`}</p>
+          <label style={{ fontWeight: 'bold' }}>Pending Approval For::</label>
```

Comment:
Small typo here. Pending Approval For:: -> Pending Approval For:, or probably just "Pending Approval:"

Reply:
> I was also wondering your thoughts on putting a count for the number of pending credits? Might make it easier for user to see when they've reached 3 (even if it's unapproved).
> 
> This might be a separate PR, but I also wonder how events are handled if they end up not being approved.

I think Henry's comment of adding credit counts to each Team Event card should handle showing them how many credits they're waiting to be approved for. 

Good question about how non-approved events are handled! @henry-li-06 what do you think?